### PR TITLE
Fixups

### PR DESCRIPTION
**Pull request by clairernovotny**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,10 +1,9 @@
 language: csharp
-mono: latest
 dotnet: 2.1.103
 
 install:
 - dotnet restore
 
 script:
- - cd reactive-extensions-test-core
+ - cd reactive-extensions-core
```
Comment (akarnokd) [inline .travis.yml]: This is either not necessary or it should be `reactive-extensions-test`.

**Comment by akarnokd** [comment]
I don't think anything above NETFramework 4.6 works on Travis at the moment. I spent hours yesterday trying to trick msbuild/dotnet to use dll's from a mono path without success: https://github.com/fluentmigrator/fluentmigrator/pull/849/files

**Comment by akarnokd** [comment]
Plus PR didn't trigger a VSTS build even though I have the triggers setup:

image

image

**Comment by clairernovotny** [comment]
There's another setting to enable forks...not sure where it is though for you. 
image

**Comment by akarnokd** [comment]
You could also try this from the [StackOverflow](https://stackoverflow.com/a/44773795/61158) answer:

```yaml
language: csharp
mono: latest
dotnet: 2.1.200
script:
  - msbuild /t:Restore
  - msbuild
  - msbuild /t:Publish /p:Configuration=Release
  - nuget install NUnit.ConsoleRunner -Version 3.8.0 -OutputDirectory testrunner
  - mono ./testrunner/NUnit.ConsoleRunner.3.8.0/tools/nunit3-console.exe ./reactive-extensions-test/bin/Release/reactive-extensions-test.dll
```

**Comment by clairernovotny** [comment]
Okay, tests pass.... my bad :P

the msbuild stuff could work to get the net461 tests working on Linux; right now it's only running the netcoreapp ones. I'm going to leave that for another time though :)

**Comment by akarnokd** [comment]
Thanks! I'll take it as it is. Now the fun part starts with VSTS. Can you see my build setups: https://akarnokd.visualstudio.com/MyFirstProject/MyFirstProject%20Team/_Build/index?_a=mine&path=%5C

**Comment by clairernovotny** [comment]
I can't see them as it's not a public project. Those are in a limited private preview for now.

**Comment by clairernovotny** [comment]
The VSTS yaml I wrote should work similarly, including publishing test results. I'd suggest using the Visual Studio 2017 if you want the tests to run the windows stuff.

You also might need this as an initial step:

```yaml
- task: DotNetCoreInstaller@0
  inputs:
    version: '2.1.200'
```

Since that'll ensure dotnet is in the path.

**Comment by akarnokd** [comment]
Ah, it's an upcoming feature: https://blogs.msdn.microsoft.com/devops/2018/04/27/vsts-public-projects-limited-preview/

It worked otherwise:

image

**Pull request merged**